apps/src/aichat/views/teacherFeedback/teacher-feedback-footer.module.scss

```diff
@@ -0,0 +1,40 @@
+@import 'color.scss';
+@import 'font.scss';
+@import '@cdo/apps/componentLibrary/typography/typography.module';
+
+.teacherFeedbackContainer {
+  display: flex;
+  align-items: center;
+  gap: 4px;
+  justify-content: flex-end;
+
+  .button i {
+    color: $light-gray-600;
+  }
+
+  .buttonAffirmative i {
+    color: $light_affirmative_500;
+  }
+
+  .buttonNegative i {
+    color: $light_negative_500;
+  }
+}
+
+.assistantFeedback {
+  flex-direction: row-reverse;
+}
+
+@keyframes fade-in {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+.tooltip {
```

Comment:
Nice, I've been wondering if we should make fade ins a global DSCO option for tooltips. cc @levadadenys

Reply:
Cool! +1 on making fade in a DSCO option. I know Mark was interested in a delay for the tooltip as well but a fade-in may address his design concern?

Reply:
Good call! I know it's something we use throughout AI Chat (styles were copied from elsewhere within AI Chat). Will defer to @levadadenys on if this is something we want to adopt in DSCO!